easybuild/easyblocks/i/icc.py

```diff
@@ -105,53 +112,117 @@ def sanity_check_step(self):
     def make_module_req_guess(self):
         """Customize paths to check and add in environment.
         """
+        debuggerpath = None
+        prefix = None
         if self.cfg['m32']:
             # 32-bit toolchain
+            libpaths = ['lib', 'lib/ia32'],
             dirmap = {
                 'PATH': ['bin', 'bin/ia32', 'tbb/bin/ia32'],
-                'LD_LIBRARY_PATH': ['lib', 'lib/ia32'],
-                'LIBRARY_PATH': ['lib', 'lib/ia32'],
+                'LD_LIBRARY_PATH': libpaths,
+                'LIBRARY_PATH': libpaths,
                 'MANPATH': ['man', 'share/man', 'man/en_US'],
                 'IDB_HOME': ['bin/intel64']
             }
         else:
-            # 64-bit toolit
-            dirmap = {
-                'PATH': ['bin', 'bin/intel64', 'tbb/bin/emt64'],
-                'LD_LIBRARY_PATH': ['lib', 'lib/intel64'],
-                'LIBRARY_PATH': ['lib', 'lib/intel64'],
-                'MANPATH': ['man', 'share/man', 'man/en_US'],
-                'IDB_HOME': ['bin/intel64']
-            }
+            # 64-bit toolkit
+            
+            # using get_software_version('GCC') won't work, while the compiler toolchain is dummy:dummy, which does not
+            # load dependencies. 
+            gccversion = get_software_version('GCC')
+            # manual approach to at least have the system version of gcc
+            if not gccversion:
```

Comment:
add a bit fat warn here. We don't really support this anymore. icc should always be installed alongside an EB GCC.